PWGHF/TableProducer/candidateSelectorLcToK0sP.cxx

```diff
@@ -164,6 +195,92 @@ struct HfCandidateSelectorLcToK0sP {
     return selectorProton.getStatusTrackBayesProbPID(track) == TrackSelectorPID::Status::PIDAccepted;
   }
 
+  template <typename T, typename U>
+  bool selectionML(const T& hfCandCascade, const U& bach)
+  {
+
+    auto candPt = hfCandCascade.pt();
+    int ptBin = findBin(binsPtML, candPt);
+    if (ptBin == -1) {
+      return false;
+    }
+
+    float inputParamsAll[24] = {
+      hfCandCascade.rSecondaryVertex(),
+      hfCandCascade.decayLength(),
+      hfCandCascade.ptProng0(),
+      hfCandCascade.ptProng1(),
+      hfCandCascade.impactParameter0(),
+      hfCandCascade.impactParameter1(),
+      hfCandCascade.v0radius(),
+      hfCandCascade.v0cosPA(),
+      hfCandCascade.mLambda(),
+      hfCandCascade.mAntiLambda(),
+      hfCandCascade.mK0Short(),
+      hfCandCascade.mGamma(),
+      o2::aod::hf_cand_casc::ctV0K0s(hfCandCascade),
+      o2::aod::hf_cand_casc::ctV0Lambda(hfCandCascade),
+      hfCandCascade.dcaV0daughters(),
+      static_cast<float>(hfCandCascade.ptV0Pos()),
+      hfCandCascade.dcapostopv(),
+      static_cast<float>(hfCandCascade.ptV0Neg()),
+      hfCandCascade.dcanegtopv(),
+      bach.tpcNSigmaPr(),
+      bach.tofNSigmaPr(),
+      hfCandCascade.cpa(),
+      o2::aod::hf_cand_3prong::ctLc(hfCandCascade),
+      o2::aod::hf_cand_3prong::yLc(hfCandCascade)};
+
+    std::vector<float> inputFeaturesF;
+    std::vector<double> inputFeaturesD;
+    uint32_t i = 1;
+    for (const auto& param : inputParamsAll) {
+      if (i & MLInputs) {
+        inputFeaturesF.push_back(param);
+        inputFeaturesD.push_back(param);
+      }
+      i = i * 2;
+    }
+
+    float scores[3] = {-1.f, -1.f, -1.f};
+    if (dataTypeML == 1) {
```

Comment:
Where are the values of `dataTypeML` documented?

Reply:
Here https://onnx.ai/onnx/api/mapping.html#l-onnx-types-mapping or also here https://github.com/microsoft/onnxruntime/blob/961fa7274ad1631bb5364ef6f836d99558b9d3b0/onnxruntime/core/optimizer/transpose_optimizer/optimizer_api.h#L46-L68

Reply:
OK, thanks. Would be quite nice, if once could use the `DataType` names directly. Would it be complicated to make the namespace visible in O2?